CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ feel free to ask us and community.
 * sqlite date hydration is susceptible to corruption ([#3949](https://github.com/typeorm/typeorm/issues/3949))
 * fixed mongodb uniques, support 3 ways to define uniques ([#3986](https://github.com/typeorm/typeorm/pull/3986))
 * fixed mongodb TTL index ([#4044](https://github.com/typeorm/typeorm/pull/4044))
+* deprecate column `readonly` option in favor of `update` and `insert` options ([#4035](https://github.com/typeorm/typeorm/pull/4035))
```

Comment:
since 0.2.17 has been released this should be moved into new section. damn hate it.

Reply:
The only thing I can think of which could minimize this is to use some kind of changelog generator, but to make it work we would have to write commit messages in specific format.
We wouldn't have to fore contributors to do so - changing messages of merge commits should be enough. Which means a bit more work when merging PRs(decide if change should be seen in changelog, and what entry should be added there).

Reply:
I guess it will be our job - change title before merging. Thats okay I think... we need to find some good changelog generator solution then.